docs/development.rst

```diff
@@ -152,26 +152,107 @@ The top-level document to open with a web browser will be
 Testing
 -------
 
-To run unit tests in the currently active Python environment, issue one of
-these example commands:
+All of the following `make` commands run the tests in the currently active
+Python environment. Depending on how the `pywbem` package is installed in that
+Python environment, either the directories in the main repository directory
+are used, or the installed package.
```

Comment:
Not sure what or the installed package means

Reply:
It refers back to the phrase "Depending on how the `pywbem` package is installed...".

Reply:
DONE - changed "the installed package" to "the installed `pywbem` package".